docs/weaver_code_anatomy.md

```diff
@@ -62,25 +59,35 @@ code: |
   if not defined("interview_metadata['a_258e_motion_for_impoundment']"):
     interview_metadata.initializeObject('a_258e_motion_for_impoundment')
   interview_metadata['a_258e_motion_for_impoundment'].update({
-    'title': '209a 258e motion for impoundment',
-    'short title': '209a 258e motion for impoundment',
-    'description': '209a 258e motion for impoundment',
-    'original_form': '',
-    'allowed courts': [
-      'Land Court',
+    "al_weaver_version": "0.82",
+    "generated on": "2021-05-28",
+    "title": "209a 258e motion for impoundment",
+    "short title": "209a 258e motion for impoundment",
+    "description": "209a 258e motion for impoundment",
+    "original_form": "",
+    "allowed courts": [
+      "Land Court",
     ],
-    'categories': [
-      'Guardianship',
+    "categories": [
+      "BE-04-00-00-00",
     ],
-    'logic block variable': 'a_258e_motion_for_impoundment',
-    'attachment block variable': 'a_258e_motion_for_impoundment_attachment',
-    'typical role': 'plaintiff',
+    "logic block variable": "a_258e_motion_for_impoundment",
+    "attachment block variable": "a_258e_motion_for_impoundment_attachment",
+    "typical role": "plaintiff",
   })
 ---
 ```
 
+1. `title`, `short title`, and `description` allow your organization's site to show more information about your form and to organize your forms more easily.
```

Comment:
I think it might be helpful to stay consistent about where the explanation comes - either before the block or after the block. I think I see what you're getting at here - listing the individual items makes more sense after the block. Maybe this can be alleviated by having a short sentence right before the block. I do think consistency in that kind of thing can help, though.

Reply:
I also think it might be useful to explain `short_title` explicitly. People don't seem to know what it is.

Reply:
Good point about the consistency. I like the short introduction, then the full code, then line by line.